pages/article/[id].js

```diff
@@ -364,6 +364,13 @@ function ArticlePage() {
       (currentUser ? true : positiveFeedbackCount - negativeFeedbackCount > 0)
   );
 
+  const replyRequestsWithCommentById = replyRequestsWithComments.filter(
```

Comment:
The new property, `replyRequest`, is in singular form. I would suggest we also return the reply request itself, instead of returning an array.

We can use [`[].find`](https://developer.mozilla.org/zh-TW/docs/Web/JavaScript/Reference/Global_Objects/Array/find) instead of filter to return the found element. It will return `null` if no such element is found.

```suggestion
  const ownReplyRequest = replyRequestsWithComments.find(
```

Reply:
ok, i will try to fix.